Doc/library/email.compat32-message.rst

```diff
@@ -660,10 +660,14 @@ Here are the methods of the :class:`Message` class:
 
       .. testsetup::
 
-         >>> from email import message_from_binary_file
-         >>> with open('Lib/test/test_email/data/msg_16.txt', 'rb') as f:
-         ...     msg = message_from_binary_file(f)
-         >>> from email.iterators import _structure
+         import email
+         from email import message_from_binary_file
+         from os.path import join, dirname
+         lib_dir = dirname(email.__file__).rstrip('/email')
```

Comment:
I made the change, thanks @bitdancer :-)  It was also Unix-like biased... Two independent errors in just one line :-O  Thank you very much for your time and patience.  These reviews are priceless :-)